docs/semgrep-app/user-management.md

```diff
@@ -1,13 +1,41 @@
 ---
-slug: role-based-access-control
+slug: user-management 
 append_help_link: true
-title: Role-based access control
-description: "This page informs users of role-based access control in Semgrep App and teaches users about roles and role management."
+title: Managing users and roles 
+description: "Learn about roles, user management, and how to implement role-based access control in Semgrep App."
 ---
 
 import MoreHelp from "/src/components/MoreHelp"
 
-# Controlling access through roles
+
+# Managing users and roles
+
+Accounts enable you to manage access to Semgrep resources, such as scans and findings, with varying levels of collaboration and visibility.
+
+Semgrep App has two types of accounts:
+
+<dl>
+    <dt>Personal account</dt>
+    <dd>Every person who signs into Semgrep App is first signed in to a <strong>personal account</strong>. In a personal account, your findings, projects, private rules, and scans are visible only to you.</dd>
+    <dt>Organization account</dt>
+    <dd>To collaborate with others, create an <strong>organization account</strong>. An organization account in Semgrep App requires an existing organization from your source code management (SCM) tool. Creating this type of account integrates Semgrep App into your organization. A user can have many organizations, provided that they are a member of that organization in their SCM.</dd>
```

Comment:
@adamkvitek

I'm going with "A user can be part of many Semgrep App organization accounts."

For the change from "An" to "The" - what's the difference? 😅

Reply:
Just that you already mentioned @s-santillan "an organization account" in the first sentence, you can make it "The organization account" in the other, because you already talked about it. :D But feel free to ignore, what you said about "A user can be part of many Semgrep App organization accounts." is good!

Reply:
Oh, I was wrong, there are more accounts, fixed my suggestion above. @s-santillan